src/components/AddServiceTile.jsx

```diff
@@ -1,10 +1,18 @@
+/**
+ * AddServiceTile component.
+ *
+ * @param {Object} props - The component props.
+ * @param {string} props.label - The label for the service tile.
+ * @param {string} [props.category] - The category of the service tile.
+ * @returns {JSX.Element} The rendered AddServiceTile component.
+ */
 import React from 'react'
 import { useClient, generateWebLink } from 'cozy-client'
 import AppLinker from 'cozy-ui/transpiled/react/AppLinker'
 
 import SquareAppIcon from 'cozy-ui/transpiled/react/SquareAppIcon'
 
-const AddServiceTile = ({ label, category }) => {
+const AddServiceTile = ({ label, category = '' }) => {
```

Comment:
With this default value, we won't go to a page with an empty category like `/discover/?type=konnector&category=`  since the condition line 18 will always be true 🤔

Reply:
empty string is considered as a falsy value in the ternary operator, but I agree it makes things a bit ambiguous when reading. I'll just use jsdoc optional param feature

Reply:
I didn't know this nuance about ternary operator thank to share it to me